sync/bundle/message/hello_test.go

```diff
@@ -15,6 +16,17 @@ func TestHelloType(t *testing.T) {
 	assert.Equal(t, m.Type(), TypeHello)
 }
 
+func TestHelloMessageMyTimeUnix(t *testing.T) {
+	myTime := time.Now()
```

Comment:
We can write a better test;
create time1 before NewHelloMessage(...)

Then time in HelloMessage should be GreaterOrEqual than `time1` and SmalerOrEqual Now.